### PR TITLE
Add links in `Community Contributors`

### PR DESCRIPTION
**Pull request by iSlodinxOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

added links

### CODE REVIEW

File: wiki/People/Community_Contributors/de.md [hunk 1]
```diff
@@ -30,15 +30,15 @@ Mitwirkende, die ihre Accounts während einer früheren Ära des Spiels registri
...
 | ::{ flag=NL }:: [statementreply](https://osu.ppy.sh/users/126198) | Ausgezeichnete Arbeit im Testen von Features und bei der Fehlerbehebung sowie Entwicklung verschiedener Funktionen für Mapper und Modder. |
-| ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) | Organisieren vieler offizieller Turniere (unter anderem dem OWC). |
-| ::{ flag=US }:: [Blazevoir](https://osu.ppy.sh/users/120265) | Wahnsinnige Moderationserfolge (entspricht etwa 6 Mitarbeitern) und fast alleinige Moderation von `#osu` während der Spitzenzeiten (9k+ Nutzer). |
+| ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) | Organisieren vieler offizieller Turniere (unter anderem dem [OWC](/wiki/Tournaments/OWC))). |
+| ::{ flag=US }:: [NaomiMinette](https://osu.ppy.sh/users/120265) | Wahnsinnige Moderationserfolge (entspricht etwa 6 Mitarbeitern) und fast alleinige Moderation von `#osu` während der Spitzenzeiten (9k+ Nutzer). |
```
Comment (Walavouchey) [inline wiki/People/Community_Contributors/de.md:34]: kinda wonder if these even should be updated. we don't usually update old user links like this on the wiki

Comment (iSlodinxOsu) [reply wiki/People/Community_Contributors/de.md]: it is done

File: wiki/People/Community_Contributors/zh.md [hunk 2]
```diff
@@ -24,21 +24,21 @@
...
+| ::{ flag=US }:: [Blazevoir](https://osu.ppy.sh/users/120265) | 难以置信的管理记录（粗略可相当于 6 位管理人员的工作量），并且几乎以一己之力在高峰时段管理 `#osu` 频道（9000 名以上用户） |
+| ::{ flag=PL }:: [Piotrekol](https://osu.ppy.sh/users/304520) | 开发了 [osu!stats](http://osustats.ppy.sh/) 以及许多作图，摸图以及打图的实用工具 |
 | ::{ flag=NO }:: [MillhioreF](https://osu.ppy.sh/users/941094) | 在 Bug 修复和新功能测试方面、以及处理用户支持请求上均有杰出的表现 |
 | ::{ flag=US }:: [DeathxShinigami](https://osu.ppy.sh/users/49516) | 3 年多默默无闻而又不辞勤劳地管理[曲包](https://osu.ppy.sh/beatmaps/packs)以及数年对月度新谱面排名竞赛的管理 |
 | ::{ flag=US }:: [LuigiHann](https://osu.ppy.sh/users/1079) | 年复一年的服务，包括了设计原创谱面皮肤，推进[故事版](/wiki/Storyboard)作为艺术和作图动力的发展 |
 | ::{ flag=CA }:: [awp](https://osu.ppy.sh/users/2650) | 早期的社区管理并且在他有限的时间过程中作出了巨大的贡献 |
 | ::{ flag=GR }:: [Sinistro](https://osu.ppy.sh/users/5530) | 作为第一位临时的社区管理员和 GMT，他为 [GMT](/wiki/People/Global_Moderation_Team) 树立了一个非常好的榜样 |
-| ::{ flag=US }:: [Ivalset](https://osu.ppy.sh/users/827) | 作为第一位行政主管和 BAT 管理员，他以一己之力从无到有建立了一个杰出的谱面管理团队 |
+| ::{ flag=US }:: [Ivalset](https://osu.ppy.sh/users/827) | 作为第一位行政主管和 [BATmanager](/wiki/People/Beatmap_Appreciation_Team/BAT_Managers) 管理员，他以一己之力从无到有建立了一个杰出的谱面管理团队 |
```
Comment (CloneWith) [inline wiki/People/Community_Contributors/zh.md:41]:
```suggestion
| ::{ flag=US }:: [Ivalset](https://osu.ppy.sh/users/827) | 作为第一位行政主管和 [BAT 管理员](/wiki/People/Beatmap_Appreciation_Team/BAT_Managers)，他以一己之力从无到有建立了一个杰出的谱面管理团队 |
```

**Review by CloneWith — COMMENTED** [review]
Didn't finish reviewing this xD

**Comment by iSlodinxOsu** [comment]
> Didn't finish reviewing this xD

it is done! 🙏

**Review by CloneWith — APPROVED** [review]
*OK for Chinese translation*

**Comment by Walavouchey** [comment]
condensed commits

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**